bindings/matrix-sdk-ffi/src/timeline/mod.rs

```diff
@@ -274,6 +274,46 @@ impl TryInto<Reply> for ReplyParameters {
     }
 }
 
+/// This type represents the "send progress" of a local event timeline item.
+#[derive(Clone, Copy, uniffi::Enum)]
+pub enum EventSendProgress {
+    /// A media is being uploaded.
+    MediaUploadProgress {
+        /// The index of the media within the transaction. A file and its
+        /// thumbnail share the same index.
+        index: u64,
```

Comment:
This should maybe be guarded with `#[cfg(feature = "unstable-msc4274")]` because it'll always be 0 for normal (non-gallery) uploads?

Reply:
Doesn't seem bad to me.